manifests/stage-ofed-driver/0400_scc.openshift.yaml

```diff
@@ -0,0 +1,47 @@
+# Please edit the object below. Lines beginning with a '#' will be ignored,
```

Comment:
I believe this will also be needed for nv-peer-mem driver as well,
IMO we should have a separate stage to deploy OpenShift's security related orbjects. WDYT ?

the alternative is having this duplicated in every stage as `stage-ofed-driver` can be opted out.